chapters/annotations.tex

```diff
@@ -120,7 +120,7 @@ \section{Annotations for Code Generation}\label{annotations-for-code-generation}
 \end{center}
 
 \begin{annotationdefinition}[Evaluate]
-\begin{synopsis}[grammar]\begin{lstlisting}
+\begin{synopsis}[grammar]\begin{lstlisting}[language=grammar]
```

Comment:
This and all the similar changes are redundant.  The language is already determined by the `synopsis` environment.